### PR TITLE
Adds a non-lethal beam rifle for sec

### PR DESCRIPTION
**Pull request by UncleBourbon**
I tried my hand at being useful and tried to add a non-lethal version of Kevinz's beam rifles

It does everything the normal beam rifle does, but a lot less murderery.
It should also spawn in the HoS's locker

### CODE REVIEW

**Comment by Steelpoint** [comment]
What does it do and sprites please.

Also someone buffing security who's name does not end in -point?

**Comment by ChangelingRain** [comment]
create a subtype!

**Comment by Bawhoppen** [comment]
clever way to remove kevinz's gun

**Comment by Repukan** [comment]
I don't get it.

Does it do damage or not?

**Comment by Shadowlight213** [comment]
This is stupid on so many levels.

File: code/modules/projectiles/guns/beam_rifle.dm [hunk 1]
```diff
@@ -47,13 +47,13 @@
 	var/wall_devastate = 0
 	var/aoe_structure_range = 1
 	var/aoe_structure_damage = 50
-	var/aoe_fire_range = 2
```
Comment (kevinz000) [inline code/modules/projectiles/guns/beam_rifle.dm]: this my friend is the rnd varient
what you want to do is make a new subtype
not make the rnd varient different

**Comment by kevinz000** [comment]
@Repukan @Shadowlight213 the joke is he's trying to add a hitscan long ranged taser to security

**Comment by kevinz000** [comment]
and i hope this is a joke

**Comment by Tacolizard** [comment]
reminds me of my first PR

**Pull request closed without merging**